webapp/channels/src/actions/views/rhs.ts

```diff
@@ -276,6 +278,11 @@ export function showChannelMembers(channelId: string, inEditingMode = false): Ac
     return async (dispatch, getState) => {
         const state = getState();
 
+        if (isPopoutWindow()) {
+            getHistory().replace(`/_popout/rhs/${getCurrentTeam(state)?.name}/${getChannelSelector(state, channelId)?.name}/channel-members`);
```

Comment:
It's easiest to hook into the existing methods to allow pop-outs to change to different ones. In this case it enables the Channel Info pop-out to become the Channel Members pop-out.

Reply:
@devinbinnie if you open the channel info popout and then click 'members' there is currently not way back. When the members rhs view is access from the channel info we need to add the `<` button in the header to navigate back to the info view. Can we keep this in the popout as well